jetty-core/jetty-http2/jetty-http2-client-transport/src/main/java/org/eclipse/jetty/http2/client/transport/internal/HttpChannelOverHTTP2.java

```diff
@@ -112,15 +112,19 @@ public void send(HttpExchange exchange)
         sender.send(exchange);
     }
 
+    void acquire()
```

Comment:
Since this method is only logging, I'd rather have it inlined.

Reply:
It matches the `release()` method just below, and it's easier to track acquire/release logs as they would be logged by the same class.